js/index.js

```diff
@@ -27,10 +62,40 @@ document.addEventListener("DOMContentLoaded", () => {
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
   }
-  
 
-  
-  /*   ///////////// MESSAGE FORM SECTION /////////////   */
+  /*
+      ====================================  
+      ************ PROJECTS  *************
+      ====================================
+   */
+  const githubRequest = new XMLHttpRequest();
+
+  githubRequest.addEventListener("load", function () {
+    if (githubRequest.readyState === 4 && githubRequest.status === 200) {
+      let repositories = JSON.parse(this.response);
+      const projectSection = document.getElementById("projects");
+      const projectList = projectSection.querySelector("ul");
+
+      for (let i = 0; i < repositories.length; i++) {
+        const project = document.createElement("li");
+        // change list to buttons that link to github
+        const repoLinks = document.createElement("button");
+        repoLinks.href = repositories[i].html_url;
```

Comment:
I don't think `<button>` supports `href`: You probably want to just make these `<a>` elements, and then style them as round rects.

Reply:
oops. Forget to check again after I switched to buttons. 

Fixed.